eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/EventDataImpl.java

```diff
@@ -133,7 +134,7 @@ public Object getObject() {
     public byte[] getBytes() {
 
         if (this.bodyData == null)
-            return null;
+            return new byte[0];
```

Comment:
I'm not sure if this is a breaking change since this method is public. Is it possible someone depends on this "null" behaviour?

Reply:
With AMQP, it is possible to send a message with no body at all (null) which is different from a body that is zero length. It is important that EventData represents the message with maximum fidelity, because sooner or later a customer will notice if it doesn't. Please leave the original behavior.